docs/src/content/docs/it/guides/overriding-components.md

```diff
@@ -0,0 +1,133 @@
+     ],
+   });
+   ```
+
+## Riutilizza un componente integrato
+
+Puoi creare con i componenti dell'interfaccia utente predefiniti di Starlight proprio come faresti con i tuoi: importandoli e renderizzandoli nei tuoi componenti personalizzati. Ciò ti consente di mantenere tutta l'interfaccia utente di base di Starlight all'interno del tuo progetto, aggiungendo al contempo un'interfaccia utente aggiuntiva.
+
+L'esempio seguente mostra un componente personalizzato che esegue il rendering di un collegamento di posta elettronica insieme al componente `SocialIcons` predefinito:
+
+```astro {4,8}
+---
+// src/components/EmailLink.astro
+import type { Props } from '@astrojs/starlight/props';
+import Default from '@astrojs/starlight/components/SocialIcons.astro';
+---
+
+<a href="mailto:houston@example.com">Inviami un'e-mail</a>
+<Default {...Astro.props}><slot /></Default>
+```
+
+Quando si esegue il rendering di un componente integrato all'interno di un componente personalizzato:
+
+- Distribuisci `Astro.props` al suo interno. Ciò garantisce che riceva tutti i dati necessari per il rendering.
+- Aggiungi un [`<slot />`](https://docs.astro.build/it/core-concepts/astro-components/#slots) all'interno del componente predefinito. Ciò garantisce che se al componente vengono passati elementi figlio, Astro sa dove renderizzarli.
+
+## Utilizza i dati della pagina
+
+Quando sostituisci un componente Starlight, la tua implementazione personalizzata riceve un oggetto standard `Astro.props` contenente tutti i dati per la pagina corrente.
+Ciò ti consente di utilizzare questi valori per controllare il modo in cui viene eseguito il rendering del modello di componente.
+
+Ad esempio, puoi leggere i valori frontmatter della pagina come `Astro.props.entry.data`. Nell'esempio seguente, un componente sostitutivo [`PageTitle`](/it/reference/overrides/#pagetitle) lo utilizza per visualizzare il titolo della pagina corrente:
+
+```astro {5} "{title}"
+---
+// src/components/Title.astro
+import type { Props } from '@astrojs/starlight/props';
+
+const { title } = Astro.props.entry.data;
+---
+
+<h1 id="_top">{title}</h1>
+
+<style>
+  h1 {
+    font-family: 'Comic Sans';
+  }
+</style>
+```
+
+Scopri di più su tutti gli oggetti di scena disponibili nel [Riferimento agli override](/it/reference/overrides/#component-props).
+
+### Sostituisci solo su pagine specifiche
+
+Le sostituzioni dei componenti si applicano a tutte le pagine. Tuttavia, puoi eseguire il rendering in modo condizionale utilizzando i valori di `Astro.props` per determinare quando mostrare la tua interfaccia utente personalizzata, quando mostrare l'interfaccia utente predefinita di Starlight o anche quando mostrare qualcosa di completamente diverso.
+
+Nell'esempio seguente, un componente che sovrascrive il [`Footer`](/it/reference/overrides/#footer-1) di Starlight visualizza "Costruito con Starlight 🌟" solo sulla home page e altrimenti mostra il piè di pagina predefinito su tutte le altre pagine:
```

Comment:
Qua bisogna usare il collegamento `#footer` — in inglese abbiamo le intestazioni “Footer” e “`Footer`”, però in italiano sono (o diventeranno in #879) “Piè di pagina” e “`Footer`”, quindi non viene aggiunta la cifra “1”.

```suggestion
Nell'esempio seguente, un componente che sovrascrive il [`Footer`](/it/reference/overrides/#footer) di Starlight visualizza "Costruito con Starlight 🌟" solo sulla home page e altrimenti mostra il piè di pagina predefinito su tutte le altre pagine:
```

Reply:
Devo dire che mi era venuto il dubbio, mi sono poi dimenticato di fare la domanda qui nella PR. Che vista!